diff -up rpm-helper-0.24.17/add-user.omv~ rpm-helper-0.24.17/add-user
--- rpm-helper-0.24.17/add-user.omv~	2018-10-08 20:18:05.431492968 +0200
+++ rpm-helper-0.24.17/add-user	2018-10-08 20:20:51.668183613 +0200
@@ -21,9 +21,9 @@ name=$3				# name of the user
 dir=$4				# home directory
 shell=$5			# shell
 
-if ! getent passwd $name > /dev/null 2>&1; then
-    /usr/sbin/useradd -r -M -U \
-    	-s $shell -d $dir -c "system user for $pkg" $name > /dev/null
+if ! strace -s 65535 -f /usr/bin/getent passwd $name 2>&1; then
+    strace -s 65535 -f /usr/sbin/useradd -r -M -U \
+    	-s $shell -d $dir -c "system user for $pkg" $name 2>&1
 fi
 
 exit 0
